### ISSUE TITLE
Test transfer of Main Image on openpeoria.nerevu.com by Upgrading CKAN after adding Main Image and Text

### ISSUE THREAD

**Issue by sottom** [openpeoria/AWS-CKAN#18]

**Comment by sottom** [comment]
Following my instructions [here](https://handbook.nerevu.com/upgrade-ckan-instance.html), I was able to transfer most everything; however, I was unable to get the result I hoped for with database transfer. Here is what went wrong.

- When I transferred the databases, my dataset groups showed up, but the groups show that there is no data in them; however, when I navigated directly to [a dataset I had before](https://openpeoria.nerevu.com/dataset/il-covid19), it was actually still present.
![image](https://user-images.githubusercontent.com/18564176/86010320-7bd73f80-b9d8-11ea-9340-50474013c908.png)

Other worries I have:

- This transfer was done between versions of CKAN that were the same (2.8.3 2020-03 version through AWS Marketplace). I feel like transferring the `templates` and `base` folders between different versions might not go as smooth because of changes the new version may have. We may have to get more granular on what to transfer (e.g. only transfer the promoted image, then change the file that references it instead of transferring the `templates` and `base` directories.

- Our new extensions don't transfer with the current instructions. We would need to transfer those and add them to the plugins variable in the CKAN config.
